Tests/SwiftFusionTests/TestUtilities.swift

```diff
@@ -0,0 +1,23 @@
+import TensorFlow
+import XCTest
+
+/// Asserts that `x` and `y` have the same shape and that their values have absolute difference
+/// less than `accuracy`.
+func assertEqual<T: TensorFlowFloatingPoint>(
+  _ x: Tensor<T>, _ y: Tensor<T>, accuracy: T, file: StaticString = #file, line: UInt = #line
+) {
+  guard x.shape == y.shape else {
```

Comment:
Can this be picked out and merged first? :)

Reply:
Nothing is stopping you :-) - new PR with cherry-pick or just the copied code

Reply:
Yes, please go ahead if you want. Or if you'd like me to do it, I could send out a PR sometime tonight. It'll probably be a few hours.

Reply:
I think that will create a merge conflict here, but I will try :)